src/Microsoft.Health.Fhir.Core/Exceptions/PartialDeleteSuccessException.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Health.Abstractions.Exceptions;
+
+namespace Microsoft.Health.Fhir.Core.Exceptions
+{
+    public class PartialDeleteSuccessException : MicrosoftHealthException
```

Comment:
In the context of an exception, I still prefer the terminology of "Incomplete" over "PartialSuccess" 🤔

Reply:
I like the name better, but I'm hesitant on Request Too Costly. We currently handle those with a 401, and I think 429 is the correct response for this. The request isn't bad, we just are unable to make it work in one pass.

Reply:
Switched and changed it to use code 413, request too large.